networkx/linalg/tests/test_algebraic_connectivity.py

```diff
@@ -46,6 +46,21 @@ def test_fiedler_vector_tracemin_unknown():
         )
 
 
+def test_spectral_bisection():
+    pytest.importorskip("scipy")
```

Comment:
Does this need scipy or numpy?  The function seems to use numpy.
```suggestion
    pytest.importorskip("numpy")
```

Reply:
Yeah but the solvers for finding `fiedler_vector` do use scipy. So without scipy this would error out too.